test/acceptance/features/steps/steps.py

```diff
@@ -189,11 +188,72 @@ def then_sbo_jq_is(context, jq_expression, sbr_name, json_value_regex):
+        print("Serverless operator is not installed, installing...")
+        assert serverless_operator.install_operator_subscription() is True, "serverless operator subscription is not installed"
+        assert serverless_operator.is_running(wait=True) is True, "serverless operator not installed"
+    context.serverless_operator = serverless_operator
+
+
+@given(u'Quarkus s2i builder image is present')
+def given_quarkus_builder_image_is_present(context):
+    """
+    Checks if quarkus s2i builder image is present
+    """
+    builder_image = BuilderImage()
+    if not builder_image.is_present():
+        print("Builder image is not present, importing and patching...")
+        assert builder_image.import_and_patch() is True, "Quarkus image import from image registry and patch failed"
+        assert builder_image.is_present() is True, "Quarkus image is not present"
+
+
+@given(u'Knative serving is running')
+def given_knative_serving_is_running(context):
+    """
+    creates a KnativeServing object to install Knative Serving using the OpenShift Serverless Operator.
+    """
+    knative_namespace = Namespace("knative-serving")
+    assert knative_namespace.create() is True, "Knative serving namespace not created"
+    assert Namespace(context.namespace.name).switch_to() is True, "Unable to switch to the context namespace"
+    knative_serving = KnativeServing(namespace=knative_namespace.name)
+    if not knative_serving.is_present():
+        print("knative serving is not present, create knative serving")
+        assert knative_serving.create() is True, "Knative serving is not created"
+        assert knative_serving.is_present() is True, "Knative serving is not present"
+
+
 # STEP
+quarkus_app_is_imported_step = u'Quarkus application "{application_name}" is imported as Knative service'
+
+
+@given(quarkus_app_is_imported_step)
+@when(quarkus_app_is_imported_step)
+def quarkus_app_is_imported_as_knative_service(context, application_name):
+    namespace = context.namespace
+    application = QuarkusApplication(application_name, namespace.name)
+    if not application.is_imported():
+        print("application is not imported, trying to import it")
+        assert application.install() is True, "Quarkus application is not installed"
+        assert application.is_imported() is True, "Quarkus application is not imported"
+    context.application = application
+
+
 @then(u'"{app_name}" deployment must contain SBR name "{sbr_name1}" and "{sbr_name2}"')
 def then_envFrom_contains(context, app_name, sbr_name1, sbr_name2):
     time.sleep(60)
     openshift = Openshift()
     result = openshift.get_deployment_envFrom_info(app_name, context.namespace.name)
     result | should.be_equal_to("[map[secretRef:map[name:binding-request-1]] map[secretRef:map[name:binding-request-2]]]")\
         .desc(f'{app_name} deployment should contain secretRef: {sbr_name1} and {sbr_name2}')
+
+
+@then(u'deployment must contain SBR name "{sbr_name}"')
```

Comment:
Well, actually it's intermediate secret name and not SBR name. 
The intermediate secret name created by SBO is same as SBR name.

But this should be corrected. :)

Reply:
done